CharlieBackend.Api.UnitTes/LessonServiceTests.cs

```diff
@@ -141,8 +140,8 @@ public async Task CreateLessonAsync()
             StudentGroup studentGroup = new StudentGroup() { Id = 3, StudentsOfStudentGroups = studentOfStudentGroup };
             StudentGroup studentGroupWrong = new StudentGroup() { Id = 100 };
 
-            DateTime lessonDate = DateTime.Parse("2021-11-18T15:00:00.384Z");
-            DateTime lessonDateWrong = DateTime.Parse("2020-11-18T15:00:00.384Z");
+            DateTime lessonDate = DateTime.Parse("2020-11-18T15:00:00.384Z");
+            DateTime lessonDateWrong = DateTime.Parse("2021-11-18T15:00:00.384Z");
```

Comment:
If I understand it correctly, for create/update lesson, `LessonDate > DateTime.Now` is considered invalid, right?
Then, judging by your changes to this line, this test will fail after 18th of December 2021